test_attitude_code/test_attitude_plotting.py

```diff
@@ -0,0 +1,106 @@
+# We use pykep for orbit determination
```

Comment:
I used a specific directory because it was mostly a quick way for me to check my code changes in a plot instead of serving as a test file which is to be merged to main. I included it in the PR for helping the review of how the model works (and I am not sure how to exclude a file in a PR). It should not be merged,  sorry for not notifying.

Reply:
You can remove it by launching this command in the same directory: "git rm name_of_the_file". But **be careful**. It will permanently delete the file. So, if you want to keep a copy of it, copy it first.  Still, we need a proper file for testing. Please include it in the repository together with the remaining checks and ask again for review once you are ready. Please also send me an email to be sure that I can react promptly.